ui/src/models/workflows.ts

```diff
@@ -803,6 +803,9 @@ export interface WorkflowList {
  * WorkflowSpec is the specification of a Workflow.
  */
 export interface WorkflowSpec {
+    activeDeadlineSeconds?: number;
+    ttlStrategy?: {};
+    podGC?: {};
```

Comment:
Could we add the docstrings to these fields as well? We've done a good job of keeping this `interface` documented so far

Reply:
done